package.json

```diff
@@ -53,4 +55,4 @@
   "lint-staged": {
     "*.{ts,json,html,tsx}": "prettier --write"
```

Comment:
here I think we should also include markdown files, users forget to keep same stlye in README.md